lib/connection.dart

```diff
@@ -60,7 +60,8 @@ class OldApiBaseAuthConnection implements IOldApiConnection {
         return await loadString(uri, post, contentType, isAuthRelated, retries + 1);
       }
 
-      addCookie(resp.headers.value("set-cookie"));
+      serverCookies.addAll(resp.cookies);
```

Comment:
I have one concern, that serverCookies never seems to be cleared? So we will be adding duplicated cookies each time we receive a response. Rather than just using the ones provided.